packages/core/types/@chainlink/index.d.ts

```diff
@@ -83,6 +83,17 @@ declare module '@chainlink/types' {
     cause: string
   }
 
+  export type AdapterErrorFull = {
```

Comment:
```suggestion
  export type AdapterErrorLog = {
```
Since we only use this for logging purposes could we name it as such?